plugin/procfs/tests/mtr/plugin.defs

```diff
@@ -0,0 +1,35 @@
+# Definition file for plugins.
```

Comment:
In 8.0 there should be no individual `plugin.defs` for new plugins - everything is put inside `mysql-test/include/plugin.defs`

Reply:
Right, right